aws/convert_types.go

```diff
@@ -311,6 +311,15 @@ func TimeValue(v *time.Time) time.Time {
 	return time.Time{}
 }
 
+// Int64TimeValue converts an int64 pointer to a time.Time value or
+// time.Time{} if the pointer is nil.
+func Int64TimeValue(v *int64) time.Time {
```

Comment:
Another approach could be to extract the highest time resolution possible (millis) and keeping the function name `Int64TimeValue`. Rounding the output to seconds feels like something that the SDK should not be directly responsible for. Thoughts?